app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,44 @@
+class QuestionsController < ApplicationController
+
+  def index
+    @questions = Question.where(user_id: current_user.id).order("created_at DESC")
+  end
+
+  def show
+    @question = Question.find params[:id]
+    @action = question_answers_path(@question)
+    @answer = Answer.new
```

Comment:
Why do you need this @action here. If it's for a form action in the view you can just put question_answers_path(@question) in the view. Hiding it behind a variable here doesn't help clarity - it just makes the reader of the view switch to another file (this one) to see what @action means.